src/directives/formioComponent.js

```diff
@@ -131,8 +131,40 @@ module.exports = [
           // Calculate value when data changes.
           if (!$scope.builder && ($scope.component.calculateValue || _get($scope.component, 'validate.json'))) {
             $scope.$watch('data', function() {
-              FormioUtils.checkCalculated($scope.component, $scope.submission, $scope.data);
-
+              if($scope.component.type !== "currency") {
+                FormioUtils.checkCalculated($scope.component, $scope.submission, $scope.data);
+              }
+              else{
+                var x = {};
```

Comment:
We need to put this logic into the FormioUtils instead of skipping over it for the currency component.

Reply:
Sure, Travis, I will make these changes in the FormioUtils